src/base32.rs

```diff
@@ -56,27 +56,21 @@ impl fmt::Display for EncodeError {
     }
 }
 
-/// Encode a u128 value to a given buffer. The provided buffer should be at least `ULID_LEN` long.
-pub fn encode_to(mut value: u128, buffer: &mut [u8]) -> Result<usize, EncodeError> {
+/// Encode a u128 value to a given buffer.
+pub fn encode_to(mut value: u128, buffer: &mut [u8; ULID_LEN]) {
```

Comment:
Can these be added as an additional method rather than replacing the existing method so that I don't need to bump a major version? We can mark the existing ones as `#[deprecated]` with a note.

Reply:
Done :+1: